README.md

```diff
@@ -171,43 +171,82 @@ Remember, every contribution counts. Let's make this project better together! 
 
 ## Set up development environment
 
-### Local setup
+### Developing dependencies
+
+- API
+  - Required packages:
+    - `golang` >= 1.22
+    - `g++`
+    - `libc-dev`
+    - `libheif` >= 1.15.1
+    - [go-face Requirements](https://github.com/Kagami/go-face#requirements) 
+        - `dlib`
+        - `libjpeg`
+        - `libblas` 
+        - `libcblas`, recommended using `libatlas-base`
+        - `liblapack`
+  - Optional tools during developing:
+    - `reflex`
+    - `sqlite`
```

Comment:
As we add these tools to the list, but mark them optional, I'd like to see a short explanation of what benefits they might add to a developer's experience: why and for what use case do we recommend using them?

Like:
  - `reflex` - a source code monitoring tool, which automatically rebuilds and restarts the server, running from the code in development: https://github.com/cespare/reflex
  - `sqlite` - the SQLite DBMS, useful to interact with Photoview's SQLite DB directly if you use it in your development environment

Reply:
Done.